frontend/packages/employee-frontend/src/components/fee-decisions/FeeDecisionFilters.tsx

```diff
@@ -140,6 +159,32 @@ function FeeDecisionFilters() {
               .getOrElse(undefined)}
             select={selectUnit}
           />
+          <Gap size="L" />
+          <FinanceDecisionManagerFilter
+            employees={employees
+              .map((e) =>
+                e.map(({ id, firstName, lastName }) => ({
+                  id,
+                  label: [firstName, lastName].join(' ')
+                }))
+              )
```

Comment:
You could do this options preprosessing when you store the value in state instead of having to do it on every render.